specification/paths/runtime/stick_table_entries.yaml

```diff
@@ -1,4 +1,38 @@
 stick_table_entries:
+  put:
+    summary: Set Entry to Stick Table
+    description: Create or update a stick-table entry in the table.
+    operationId: setStickTableEntries
+    tags:
+      - StickTable
+    parameters:
+      - description: Stick table entry
+        name: stick_table_entry
+        in: body
+        schema:
+          type: object
+          required:
+            - stick_table
+            - process
+            - key
+            - value
+            - data_type
+          properties:
+            stick_table:
+              type: string
+            process:
+              type: integer
```

Comment:
both of those can be a qs parameter, as they are hierarchial, while the rest of the fields represent the actual value.

Reply:
Yea but maybe a better practice to keep all inputs in the same env (body or qs)? :thinking:

Reply:
I would still like to follow the same inputs we have in e.g. GET method. For process and stick_table parameters we are using them either as a filter (process) or as a part of the identifier (stick_table + key), and looking at this, we should move this to a PUT at `/services/haproxy/runtime/stick_tables_entries/{key}` instead of `/services/haproxy/runtime/stick_tables_entries/` as this is a better practice, we are changing the resource with that key at that stick_table so all will be in the URL, the process is a filter thing, that should be deprecated soon enough.

Reply:
@mjuraga PTAL